vue/components/ui/generic-button-platforme/generic-button-platforme.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <button class="btn btn-content" v-bind:style="buttonStyleColor">
+        {{ buttonText }}
+    </button>
+</template>
+
+<style scoped>
+.btn-content:hover:not(:disabled)::after {
+    right: 4px;
+}
+
+.btn-content::after {
+    background-image: url(https://d3moonnr9fkxfg.cloudfront.net/royal-canin.sprite--brand3--xs.svg?v=8-7-8);
+    background-position: 45.45% 85.71%;

```

Comment:
Can you use `px` instead? I know the result is not quite the same but a fixed position is probably enough.